apistructs/pipeline_label.go

```diff
@@ -52,3 +56,32 @@ type TargetIDSelectByLabelRequest struct {
 	// OrderByTargetIDAsc 根据 target_id 升序，默认为 false，即降序
 	OrderByTargetIDAsc bool `json:"orderByTargetIDAsc"`
 }
+
+type PipelineLabelBatchInsertRequest struct {
+	Labels []PipelineLabel `json:"labels"`
+}
+
+type PipelineLabelListRequest struct {
+	PipelineSource  PipelineSource `schema:"pipelineSource" json:"pipelineSource"`
+	PipelineYmlName string         `schema:"pipelineYmlName" json:"pipelineYmlName"`
+	TargetIDs       []uint64       `schema:"targetIds" json:"targetIds"`
```

Comment:
In tag you should use schema:"targetID", and invoker will use this query params like: /api/xxx?targetID=1&targetID=2

use ID, not Id.

Reply:
```golang
TargetIDs       []uint64       `schema:"targetID"`
```

No need json tag.

fdp client: /api/pipeline-labels?targetID=1&targetID=2&targetID=3

Reply:
https://stackoverflow.com/questions/11944410/passing-array-in-get-for-a-rest-call
use targetIds not a bad idea, user see end with s then he will know he can send a array params

Reply:
add json tag does not affect use